pkg/apis/openebs.io/v1alpha1/cas_pool.go

```diff
@@ -17,6 +17,29 @@ package v1alpha1
 
 // CasPool is a type which will be utilised by CAS engine to perform
 // storagepool related operation
+type CasPoolKey string
+type CasPoolVals int
+
+const (
+	// HostNameCPK is the kubernetes host name label
+	HostNameCPK CasPoolKey = "kubernetes.io/hostname"
+	// StoragePoolClaimCPK is the storage pool claim label
+	StoragePoolClaimCPK CasPoolKey = "openebs.io/storage-pool-claim"
+	// DiskTypeCPK is the node-disk-manager disk type e.g. 'sparse' or 'disk'
+	DiskTypeCPK CasPoolKey = "ndm.io/disk-type"
+	// PoolTypeMirroredCPK is a key for mirrored for pool
+	PoolTypeMirroredCPK CasPoolKey = "mirrored"
+	// PoolTypeMirroredCPK is a key for striped for pool
+	PoolTypeStripedCPK CasPoolKey = "striped"
+	// TypeSparseCPK is a key for sparse disk pool
+	TypeSparseCPK CasPoolKey = "sparse"
+	// TypeDiskCPK is a key for physical disk pool
+	TypeDiskCPK CasPoolKey = "disk"
```

Comment:
There is a constant named DiskTypeCPK and this one is TypeDiskCPK. Need better names. Any suggestions @kmova @AmitKumarDas ?